go/api/glide_cluster_client.go

```diff
@@ -1384,3 +1384,79 @@ func (client *GlideClusterClient) FCallReadOnlyWithArgsWithRoute(
 func (client *GlideClusterClient) FCallReadOnlyWithArgs(function string, args []string) (ClusterValue[any], error) {
 	return client.FCallReadOnlyWithArgsWithRoute(function, args, options.RouteOption{})
 }
+
+// FunctionStats returns information about the function that's currently running and information about the
+// available execution engines.
+// The command will be routed to all nodes by default.
+//
+// Since:
+//
+//	Valkey 7.0 and above.
+//
+// See [valkey.io] for details.
+//
+// Return value:
+//
+//	A [ClusterValue] containing a map of node addresses to their function statistics.
+//
+// [valkey.io]: https://valkey.io/commands/function-stats/
+func (client *GlideClusterClient) FunctionStats() (
+	ClusterValue[FunctionStatsResult], error,
```

Comment:
```suggestion
	map[string]FunctionStatsResult, error,
```
Since it is always a map
and copy docs from standalone